docs/audits/index.md

```diff
@@ -0,0 +1,24 @@
+# Jupyter Accessibility Audits
+
+This is the compendium of accessibility audits in the Jupyter ecosystem. Each
+one of these identifies the current state of the interface regarding a given
+accessibility tool, area or topic as well as improvement strategies in order
+to fix shortcomings and/or failures on each of the topics.
+
+The aim of the findings is to determine the improvements needed for disabled
+users to avoid complicated strategies and workflows that may prevent them to
+use the software in its 100%.
+
+## Audits
+```{toctree}
+:maxdepth: 1
+
+zoom-audit
+keyboard-audit
+```
+
+## See also
+
+- [Notebook v7.0.0 for WCAG 2.1 audit](https://github.com/jupyter/notebook/issues/6800)
+- [Screen reader with keyboard shortcuts audit](https://github.com/jupyterlab/jupyterlab/issues/6573)
+- [Issues in JupyterLab v2.2.6 for WCAG 2.1 audit](https://github.com/jupyterlab/jupyterlab/issues/9399)
```

Comment:
Links should follow a similar pattern - i.e., which tool do they refer to? As a reader I do not get enough information from the current wording.


```suggestion
- [Notebook - v7.0.0 WCAG 2.1 audit (March 2023)](https://github.com/jupyter/notebook/issues/6800)
- [JupyterLab - Screen reader with keyboard shortcuts audit (June 2019)](https://github.com/jupyterlab/jupyterlab/issues/6573)
- [JupyterLab -  v2.2.6 WCAG 2.1 audit (November 2021)](https://github.com/jupyterlab/jupyterlab/issues/9399)
```